src/components/material.js

```diff
@@ -81,13 +81,16 @@ module.exports.Component = registerComponent('material', {
 
   updateShader: function (shaderName) {
     var data = this.data;
-    var Shader = shaders[shaderName] && shaders[shaderName].Shader;
-    var material;
-    if (!Shader) { throw new Error('Unknown shader ' + shaderName); }
-    this.shader = new Shader();
-    this.shader.el = this.el;
-    material = this.shader.init(data);
-    this.setMaterial(material);
+    var ShaderClass = shaders[shaderName] && shaders[shaderName].Shader;
```

Comment:
Is there another word we can use? Because there's `shaders`, `.Shader`, `this.shader`, `shader`.